nixos/tests/wireguard/default.nix

```diff
@@ -13,6 +13,7 @@ let
     wg-quick = callTest ./wg-quick.nix;
     wg-quick-nftables = args: callTest ./wg-quick.nix ({ nftables = true; } // args);
     generated = callTest ./generated.nix;
+    file-config = callTest ./file-config.nix;
```

Comment:
What about using a more specific file name since there are other *File options in this module?

Reply:
This test should handle both of the file options -- I saw that existing tests didn't test either file option in exclusion, so I added a test to cover "write a file to environment.etc and reference it" for both the private key and the new hostname file options.

Reply:
Oh, I did not notice you test other *File options. Feel free to mark this as resolved.